### PR TITLE
Javascript conditionals - Dylan Dao

### PR DESCRIPTION
**Pull request by dilpills**
Screenshot 2023-03-22 at 2 56 44 PM (2)
Screenshot 2023-03-22 at 2 31 41 PM (2)
Screenshot 2023-03-22 at 2 24 10 PM (2)

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    case 'horror':
+      return 'The Ring';
+    case 'drama':
+      return 'Awaken';
+    case 'musical':
+      return 'High School Musical';
+    case 'sci-fi':
+      return 'The Mandalorian';
+    default:
+      return 'try again bud';
```
Comment (shawnkost) [inline javascript-conditionals/main.js:83]: Funny, but we want to inform the user where they went wrong if they passed an incorrect value! Take a look at the error message I gave, and use the same one.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just one small change requested Dylan!

File: javascript-conditionals/main.js (hunk 1 above)
Comment (dilpills) [reply javascript-conditionals/main.js]: updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**